druntime/src/core/internal/gc/impl/conservative/gc.d

```diff
@@ -3003,7 +3007,14 @@ struct Gcx
                                     sentinel_Invariant(q);
 
                                     if (pool.finals.nbits && pool.finals.test(biti))
-                                        rt_finalizeFromGC(q, sentinel_size(q, size), pool.getBits(biti));
+                                    {
```

Comment:
this level of indentation is a bit silly. You could make this `if (pool.mark.test(biti)) continue;`.

(I note that the `if (bin < Bins.B_PAGE)` way up file could also be turned into an early continue, but that can probably wait for a separate refactoring PR)

Reply:
Fixed.